### ISSUE TITLE
Documentation for test(), command(), skip() rule types needs to be more verbose

### ISSUE THREAD

**Issue by jottinger** [mpe85/grampa#47]

**Comment by mpe85** [comment]
@jottinger, I will have a look at it shortly.

Something specific in your mind that should be added to the docs?

**Comment by jottinger** [comment]
I think *any* descriptive documentation would help. The javadocs are very sparse; the wiki doesn't mention them *at all*, and the test cases don't do anything more than exercise the functions as far as I can see.

**Comment by mpe85** [comment]
Did you have a look at the action rules section in the wiki? https://github.com/mpe85/grampa/wiki/Rules#action-rules

**Comment by jottinger** [comment]
Yes, and it needs to be more verbose, IMO. You describe what they *do* fairly appropriately, but that's pretty much it. What I'd like to see is strategies to use those to handle different types, accumulators unrelated to the grammar's stack type, when they should be used, how to ignore them for test conditions, more things like that.

**Comment by mpe85** [comment]
OK, I see. I will try to revisit/revise the docs when I have a bit more time.
